lib/gitlab/client/projects.rb

```diff
@@ -421,10 +421,12 @@ def create_fork(id, options={})
     #   Gitlab.edit_project(42, { name: 'project_name' })
     #
     # @param  [Integer, String] project The ID or name of a project.
-    # @param  [Hash] options A customizable set of options.
+    # @param  [Hash] options A customizable set of options
     # @option options [String] :name The name of a project
-    # @option options [String] :path The name of a project
-    # @option options [String] :description The name of a project
+    # @option options [String] :path The URL-safe path of the project (used in Gitlab URLs)
```

Comment:
Since we have a `url_encode` method should we just encode the `:path` for them if it's been provided?